File: solutions/0033-search-in-rotated-sorted-array/solution.php
class Solution {
    function search($nums, $target) {
        $r=-1;
        for($i=0;$i< count($nums);$i++){
            if($nums[$i]==$target){
                return $i;
            }
            else{
                $r == -1;
            }
            
        }
        return $r;
    
    }
}

File: solutions/0033-search-in-rotated-sorted-array/solution.c
int search(int* nums, int numsSize, int target) {
    int r=0;
    for(int i=0;i<numsSize;i++){
        if(nums[i]==target){
            return i;
        }
        else{
            r=-1;
        }
    }
    return r;
    
}
